### DISCUSSION TITLE
(an idea)   ALT+TAB cycle further options: include (optionally) other workspaces

### DISCUSSION THREAD

**Discussion by peterjosvai** [linuxmint/discussions#595] [Ideas]
hi, 

It would be great if we could cycle through other workspaces' windows too when using ALT + TAB...

Of course, this is a long established, "core" functionality, which is like one of the the basic words in a mother language, 
and changing it could  easily cause irritation... 

BUT... what I imagine is using an additional key, optionally, when you want to break out of the workspace's box...

For example, you're cycling through the windows....
and you realize that what you want to "bring up" is in another workspace, 
Then you could just (for example) hit the CTRL button (or UP-arrow) ... which would trigger
showing other workspaces' window icons... 
in separate "rows" below (and above) the one you're "using"....

Say, you have 3 workspaces...
you're in WS #2.... 
you're  ALT + TAB-ing... 
and then you press CTRL (**or anything**, shift, for example), 
two more rows of windows would be visible... 
workspace #1 and workspace #2

now with the UP and DOWN arrows we can jump over to the other workspace.... 
and select the application window we seek...

THIS COULD be sophisticated....
but the point is to get a picture of the basic usage... I guess...

ONE note: since it would take pressing an extra button to get this functionality, 
most people wouldn't even know it exists :) 
which is a good thing...
cause no irritation would happen...
only people who want to use this would see more workspaces in that ALT + TAB screen....

Peter

- - - - thank you for developing Linux Mint - - - -

PS:
I did a search:  "alt tab cycle"

**Comment by anaximeno** [comment]
> It would be great if we could cycle through other workspaces' windows too when using ALT + TAB...

Isn't this already possible?

In the windows settings at the alt-tab section you can enable the option to show up windows from all workspaces.

**Reply by peterjosvai** [reply]
hi, 

indeed so!

I apologize that I never have realized that we had this option...

BUT...
as of Cinnamon 6.2.9, all window icons are in one row... which kinda sucks.. meaning: it can be improved!

I'll post it as an idea independently, for clarity's sake...

**thanks again!**

I'll close this one...
